### ISSUE TITLE
Please add event type for CALdav calendars

### ISSUE THREAD

**Issue by sphh** [SimpleMobileTools/Simple-Calendar#653]
I can add an event type for the local calendar, but not for a caldav synced calendar. The corresponding action is simply missing.

Do you think it's possible to add this feature? Thanks.

**Comment by tibbi** [comment]
not really, CalDAV does not support types. You have to pick some other calendar. That is basically a type.

**Comment by sphh** [comment]
Well, I might have misnamed it... CalDAV supports CATEGORIES, which are e.g. used by Thunderbird/Lightning to colour events. Can you think of adding support for categories?

**Comment by tibbi** [comment]
not sure, Ill take a look at it in the future

**Comment by ElpyDE** [comment]
Plus one for adding categories support to this awesome app. Right now it's the only feature that I'm really missing.

**Comment by davidhedlund** [comment]
The title is
> Please add CATEGORIES for CalDAV calendars

I guess it should be renamed to
> Please add CATEGORIES for events

Because 1) the categories are intended for events not calendars, right? 2) Categories can also be added to regular iCalendar (.ical) events.

![Screenshot at 2019-04-14 08:37:48](https://user-images.githubusercontent.com/6416073/56089274-9c2ce400-5e90-11e9-89a4-a003f716bc27.png)

**Comment by ElpyDE** [comment]
@davidhedlund That's probably true. Guess/hope @tibbi also understood that this is what was asked for.

**Comment by tibbi** [comment]
you already have them, it is just called Event Type. You can set it only at locally stored events, not caldav ones

**Comment by davidhedlund** [comment]
> you already have them, it is just called Event Type. You can set it only at locally stored events, not caldav ones

Thanks. Shouldn't the title be changed from

>Please add CATEGORIES for CalDAV calendars

to

>Please add Event Type for CalDAV calendars

?

**Comment by tibbi** [comment]
it doesnt matter what the title is, everyone knows what is it about. It is explained in the reporters post already

**Comment by moshpirit** [comment]
Is there a way to sort by event types? for example, imaging I have two big event types:
1. Everyday actions (such as cooking etc)
2. Big meetings, exams, etc 

I would like to be shown the big events instead of by just chronological order. Would it be hard to implement? (I'm not a developer so I have no idea about these difficulties 😅)

Thanks a lot for the app!! It's gorgeous!! :heart:

**Comment by tibbi** [comment]
you cannot sort like that. Try setting the big events as all-day ones, they will be shown first

**Comment by moshpirit** [comment]
I was asking if there's any way for this feature to be developed 😅 sorry if I didn't explained myself correctly

**Comment by tibbi** [comment]
not really, so far nobody requested sorting by event type. Anyway, this issue isnt about that.

**Comment by moshpirit** [comment]
Well, if categories are event types, sorting by event types is a way to difference by categories in terms of importance. This is why I thought this was on topic. Sorry if it wasn't

**Comment by rfc2822** [comment]
The AOSP ActiveSync Exchange adapter and DAVx⁵ use extended properties of events to store categories in this format: https://www.davx5.com/manual/technical_information.html#categories

It could be useful if Simple Calendar was able to view/edit these categories.

**Comment by sparagi** [comment]
I just noticed that version 6.21.7 Pro from F-Droid includes support for per-event colors.  Since I can set them in Simple Calendar and then see them in Nextcloud Calendar's web UI, I suspect they have been implemented using CATEGORIES.  I had to turn on some settings in DAVx5 to see the UI in SC; more information in #360.

Thank you very much, kind developers!

**Comment by K4LCIFER** [comment]
It would be great if simple calendar supported CATEGORIES; they are very useful.
